cypress/support/generateUser.js

```diff
@@ -0,0 +1,11 @@
+function generateUser() {
+  const user = {
```

Comment:
Hardcoded user information is not a good practice. You can use a library like faker.js to generate unique user data
https://fakerjs.dev/